apps/client/src/widgets/mypage/components/account-menu-bar/account-menu-bar.css.ts

```diff
@@ -0,0 +1,24 @@
+import { style } from '@vanilla-extract/css';
+
+import { themeVars } from '@bds/ui/styles';
+
+export const container = style({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  gap: '1rem',
+
+  position: 'sticky',
+  bottom: 0,
+  width: '100%',
+  height: '5.8rem',
+
+  padding: '1rem',
+  backgroundColor: themeVars.color.white,
+});
+
+export const leftButtonWrapper = style({
+  display: 'flex',
+  paddingRight: '1rem',
+  borderRight: `1px solid ${themeVars.color.gray400}`,
+});
```

Comment:
```suggestion
export const verticalLine = style({
  width: '0.1rem',
  height: '2.5rem',
  backgroundColor: themeVars.color.gray400,
});
```

제 취향대로 한다면 css도 이렇게 바뀔 것 같아요
(근데 진짜 just 취향)